import_flow_events.py

```diff
@@ -5,7 +5,7 @@
 import import_events
 
 TEMPORARY_SCHEMA = """
-    type VARCHAR(64) NOT NULL ENCODE zstd,
+    type VARCHAR(79) NOT NULL ENCODE zstd,
```

Comment:
I've added a comment.

The event is `flow.continued.${flow_id}` so 15 + 64 = 79 characters. I briefly considered making it 80 characters on account of that being a round number, but it's only a round number if you're a ten-fingered weirdo from planet earth. Whereas 79 is prime in the universal language of maths, which makes it a round number EVERYWHERE. :smile: